app/dashboard/templates/shared/zero_balance_error.html

```diff
@@ -22,13 +22,10 @@ <h3>{% trans "You're almost there!" %}</h3>
       <ol class="pl-3 mb-0 font-smaller-1">
         <li>{% trans "In order to continue, you'll need some Ether." %}</li>
         <li>{% trans "'Ether' is a digital asset that can be used to pay for the computational resources needed to run an application or program. In this case, Gitcoin." %}</li>
-        <li>{% trans "The easiest way to get Ether is to request it via the Gitcoin Faucet:" %}</li>
```

Comment:
wouldn't it be nice to point to a trusted faucet website? e.g https://faucet.metamask.io/